packages/template-retail-react-app/app/pages/reset-password/index.jsx

```diff
@@ -68,41 +51,14 @@ const ResetPassword = () => {
                 marginBottom={8}
                 borderRadius="base"
             >
-                {!showSubmittedSuccess ? (
+                {path === '/reset-password-landing' ? (
```

Comment:
Should this be a constant? Should this string match with the `magicLink` in the `ssr.js`?

Reply:
Good point yes it should. Making the constant makes sense so that both this file and ssr.js file can point to the same path